### PR TITLE
Add docs for query breakdown output of explain analyze

### PR DESCRIPTION
**Pull request by mfussenegger**

### CODE REVIEW

File: blackbox/docs/sql/statements/explain.rst [hunk 1]
```diff
@@ -36,6 +36,78 @@ timings of the different phases of the plan are returned.
    in-depth details.
 
 
+The output of ``EXPLAIN ANALYZE`` also includes a break down of the query
+execution if the statement being explained involves queries which are executed
+using ``Lucene``.
```
Comment (chaudum) [inline blackbox/docs/sql/statements/explain.rst]: Not sure if "Lucene" should be formatted as inline code.

File: blackbox/docs/sql/statements/explain.rst [hunk 2]
```diff
@@ -36,6 +36,78 @@ timings of the different phases of the plan are returned.
...
+| ``build_scorer``                  | A ``Scorer`` object is used to    |
+|                                   | iterate over documents matching   |
+|                                   | the query and generate scores for |
+|                                   | them. Note that this includes     |
+|                                   | only the time to create the       |
+|                                   | scorer, not that actual time      |
+|                                   | spent on the iteration.           |
++-----------------------------------+-----------------------------------+
+| ``score``                         | Shows the time it takes to score  |
+|                                   | a particular document via it’s    |
```
Comment (chaudum) [inline blackbox/docs/sql/statements/explain.rst]: its

**Comment by mfussenegger** [comment]
retest this please

**Comment by mfussenegger** [comment]
retest this please

**Pull request merged**